projects/projects.json

```
@@ -119,7 +119,7 @@
  "logo": "https://mscorpnews.blob.core.windows.net/ncmedia/2014/10/MSFT_logo_rgb_C-Gray.png",
  "web" : "http://www.microsoft.com"},
  {"name": "Pivotal Software, Inc.",
-  "logo": "https://brandfolder.com/pivotal-assets/attachments/ohxttd-7yxa60-clvf50/pivotal-assets-pivotal-genericfile.png",
+  "logo": "http://steeltoe-site-content.s3.amazonaws.com/images/Pivotal%20Logo.png",
```

```suggestion
  "logo": "https://steeltoe-site-content.s3.amazonaws.com/images/Pivotal%20Logo.png",
```

HTTPS would be better.